.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
You usually avoid committing editor files to the repo.

Reply:
Okey!